storage/store/blocks.go

```diff
@@ -0,0 +1,109 @@
+package store
```

Comment:
Unfortunately, github can’t recognize the file moves from badger folder to the store folder, as well as many files in the storage/badger/operation folder. I tried to revert and add back, but still didn’t work. We have to clone the repo in a different folder, and compare the files on master in your local.